cli/tests/repl_tests.rs

```diff
@@ -897,4 +897,50 @@ mod repl {
     assert_ends_with!(out, "\"done\"\n");
     assert!(err.is_empty());
   }
+
+  #[test]
+  fn npm_packages() {
+    let env_vars = vec![
+      ("DENO_NODE_COMPAT_URL".to_string(), util::std_file_url()),
+      ("DENO_NPM_REGISTRY".to_string(), util::npm_registry_url()),
+      ("NO_COLOR".to_string(), "1".to_string()),
+      (
+        "DENO_UNSTABLE_NPM_SYNC_DOWNLOAD".to_string(),
+        "1".to_string(),
+      ),
+    ];
+
+    {
+      let (out, err) = util::run_and_collect_output_with_args(
+        true,
+        vec!["repl", "--quiet", "--allow-read", "--allow-env"],
+        Some(vec![
+          r#"import chalk from "npm:chalk";"#,
+          "chalk.red('hello')",
```

Comment:
Maybe do something like `'hel' + 'lo'` so that the assertion below won't be done accidentally on the input.